src/common/Utilities/Util.cpp

```diff
@@ -469,6 +469,11 @@ void wstrToUpper(std::wstring& str)
     std::transform(str.begin(), str.end(), str.begin(), wcharToUpper);
 }
 
+void strToLower(std::string& str)
+{
+    std::transform(str.begin(), str.end(), str.begin(), [](char c) { return std::tolower(c); });
```

Comment:
@Shauren do you think `strToLower(std::string& str)` should internally convert to wstr and back?